pages/api/v1/posts/summary.ts

```diff
@@ -46,7 +46,9 @@ const handler: NextApiHandler<IPostSummaryResponse | MessageType> = async (req,
 	const contentToSummarize = postData?.content || postData?.summary;
 
 	if (!contentToSummarize) {
-		return res.status(200).json({ summary: '' });
+		const noSummary = 'No summary available';
```

Comment:
- get from subsquare, 
- save content from subsquare to our db
- generate summary on that content
- if not content found: no summary copy return

Reply:
Copy: No contextual information available for this post.